src/webview/css/css.js

```diff
@@ -32,13 +32,22 @@ code {
   white-space: pre-wrap;
   padding: 0 0.25em;
 }
+code::before {
+  /* Add a left-to-right embedding character before each code block */
```

Comment:
I tested with these removed and the `code` blocks still do RTL correctly, I guess the `dir="auto"` does the job.